services/common/src/components/project/ProjectDescriptionTab.tsx

```diff
@@ -0,0 +1,472 @@
+import React, { useEffect, useState } from "react";
+import { Row, Col, Typography, Button, Alert, Badge } from "antd";
+import Callout from "@mds/common/components/common/Callout";
+import { CALLOUT_SEVERITY } from "@mds/common/constants/strings";
+import CoreTable from "@mds/common/components/common/CoreTable";
+import { getProject } from "@mds/common/redux/selectors/projectSelectors";
+import { useSelector, useDispatch } from "react-redux";
+
+import { getPermits } from "@mds/common/redux/selectors/permitSelectors";
+import { renderTextColumn } from "@mds/common/components/common/CoreTableCommonColumns";
+import { IAuthorizationSummary } from "@mds/common/interfaces";
+import { useHistory } from "react-router-dom";
+
+import {
+  getDropdownProjectSummaryPermitTypes,
+  getProjectSummaryAuthorizationTypesArray,
+  getTransformedProjectSummaryAuthorizationTypes,
+} from "@mds/common/redux/selectors/staticContentSelectors";
+import { ColumnsType } from "antd/es/table";
+
+import { formatDateTimeTz } from "@mds/common/redux/utils/helpers";
+
+import { PresetStatusColorType } from "antd/es/_util/colors";
+
+import {
+  updateProjectSummary,
+  fetchProjectById,
+} from "@mds/common/redux/actionCreators/projectActionCreator";
+import { isArray } from "lodash";
+import { removeNullValuesRecursive } from "@mds/common/constants/utils";
+import Loading from "@mds/common/components/common/Loading";
+
+export const ProjectDescriptionTab = () => {
```

Comment:
Just noticed this- there's `export const ProjectDescriptionTab` up here, and then `export default ProjectDescriptionTab` at the bottom. Don't need both!
I'd go with the export at the bottom so you can import it like `import ProjectDescriptionTab from <path>`

Reply:
The export at the top has been removed now.